annotations/knative-annotations/src/main/java/io/dekorate/knative/manifest/KnativeManifestGenerator.java

```diff
@@ -211,25 +212,25 @@ public void generate(KnativeConfig config) {
 
     // Global autoscaling configuration
     if (config.getGlobalAutoScaling() != null) {
-      if (!isDefault(config.getGlobalAutoScaling())) {
+      if (config.getGlobalAutoScaling().getAutoScalerClass() != null
+          && config.getGlobalAutoScaling().getAutoScalerClass() != AutoScalerClass.kpa) {
         resourceRegistry.decorate(KNATIVE, new AddConfigMapResourceProvidingDecorator(CONFIG_AUTOSCALER, KNATIVE_SERVING));
```

Comment:
Before these changes, we were creating the `config-autoscaler` configmap, when the property `target-utlization-percentage` is set which does not use it. 

I've removed the `isDefault` method and create the configmap if and only if it's necessary.